src/components/PromiseList/PromiseList.svelte

```diff
@@ -87,8 +87,17 @@
 		</div>
 	{/if}
 	<div class="{isList ? 'flex flex-col gap-[2px]' : 'grid gap-6 md:grid-cols-3'} mx-auto w-fit">
-		{#each sortedData as promiseSummary, index (index)}
-			<PromiseCard {promiseSummary} {isList} />
+		{#each sortedData as { id, status, latestProgressDate, party, statements, keywords, categories }, index (index)}
+			<PromiseCard
+				{id}
+				{status}
+				{latestProgressDate}
+				{party}
+				{statements}
+				{keywords}
+				{categories}
+				{isList}
+			/>
```

Comment:
Tips: You can spread object in Svelte as props like

```svelte
{#each sortedData as promiseSummary, index (index)}
  <PromiseCard {...promiseSummary} {isList} />
{/each}
```

Which would have the same result but more concise